librabbitmq/amqp_openssl.c

```diff
@@ -26,6 +26,7 @@
 #include <openssl/engine.h>
 #include <openssl/err.h>
 #include <openssl/ssl.h>
+#include <openssl/rsa.h>
```

Comment:
clang-format wants this include to be between line 27 and line 28 (includes are sorted alphabetically).

Reply:
Fixed